src/VisualStudio/Core/Next/FindReferences/StreamingFindUsagesPresenter.TableDataSourceFindUsagesContext.cs

```diff
@@ -197,12 +198,23 @@ public IDisposable Subscribe(ITableDataSink sink)
 
             #region FindUsagesContext overrides.
 
-            public override void SetSearchLabel(string displayName)
+            public override void SetSearchTitle(string title)
             {
-                var labelProperty = _findReferencesWindow.GetType().GetProperty("Label");
-                if (labelProperty != null)
+                try
+                {
+                    // Editor renamed their property from Label to Title, and made it public.
```

Comment:
Consider having fallback code for people who haven't updated yet that tries `Label`?